packages/@aws-cdk/aws-stepfunctions-tasks/test/sns/publish.test.ts

```diff
@@ -38,6 +38,61 @@ describe('Publish', () => {
       },
     });
   });
+  test('with message attributes', () => {
+    // GIVEN
+    const stack = new cdk.Stack();
+    const topic = new sns.Topic(stack, 'Topic');
+
+    // WHEN
+    const task = new SnsPublish(stack, 'Publish', {
+      topic,
+      message: sfn.TaskInput.fromText('Publish this message'),
+      messageAttributes: [
+        {
+          name: 'cake',
+          type: SnsMessageAttributeType.STRING,
```

Comment:
don't think this is needed in this test since that's the default right?

Reply:
true - we can remove it.